include/triton/Analysis/Utility.h

```diff
@@ -3,19 +3,20 @@
 
 #include "mlir/Analysis/DataFlowFramework.h"
 #include "mlir/Analysis/SliceAnalysis.h"
+#include "mlir/Dialect/ControlFlow/IR/ControlFlowOps.h"
```

Comment:
Why do we need to include "ControlFlowOps.h" in "Utility.h"? I don't see it's being used.

Reply:
let me remove this.